po/Makevars

```diff
@@ -0,0 +1,78 @@
+# Makefile variables for PO directory in any package using GNU gettext.
+
+# Usually the message domain is the same as the package name.
+DOMAIN = udisk2
```

Comment:
Nice typo, took us a while to find out why the tests are failing... :-)

Reply:
Ouch, nice catch, sorry about that...